### DISCUSSION TITLE
Issue with panics

### DISCUSSION THREAD

**Discussion by TheTechRobo** [rust-wii/ogc-rs#104] [Q&A] [answered]
When I panic in my program, even after `Console` and `Video` are set up (`println!` works fine), I just see the (libogc?) exception message, not the actual panic message.

![image](https://github.com/rust-wii/ogc-rs/assets/52163910/6a880a9b-af12-464f-a1b9-a16b3c081d5d)

Maybe this is because the panic handler [uses `abort`](https://github.com/rust-wii/ogc-rs/blob/1b17de75cd10c9ed5e4912edababfca1119da0fe/src/runtime.rs#L49)? Not sure.

Additionally, is there any way to put in a custom panic handler (disable ogc-rs's) without having to modify the crate?

Thanks in advance!

**Comment by ProfElements** [comment]
Abort causes a program exception you are right. Right now there is not way to change the panic handler.

**Reply by ProfElements** [reply, marked as answer]
fade26a should fix the aborting issue. and if you disable the default features you should be able to provide your own.

**Reply by TheTechRobo** [reply]
Awesome, thank you!
